jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Carousel.java

```diff
@@ -35,7 +35,8 @@
 	protected String indicatorsLocator = "li[data-slide-to]";
 
 	@UI(".carousel-item") @GetAny public DataList<T, D> slides;
-	@UI(".carousel-item") @VisualCheck UIElement activeSlide;
+	@UI(".carousel-item") @VisualCheck
+    UIElement activeSlide;
```

Comment:
Do you really need this changes? The same for jdi-light-html-tests/src/main/java/pseudo/site/forms/CustomForm.java

Reply:
extra newLine removed